src/main/java/org/terracotta/statistics/LongSample.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * All content copyright Terracotta, Inc., unless otherwise indicated.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terracotta.statistics;
+
+import java.io.Serializable;
+
+/**
+ * @author Mathieu Carbou
+ */
+public class LongSample implements Serializable, Comparable<LongSample> {
```

Comment:
Ok so LongSample is related to this commit, this is new, but I've fixed `Sample` because I saw missing things and errors (timestamp). I will put this in another commit